.gitignore

```diff
@@ -113,6 +113,7 @@ coverage/
 # Generated artifacts
 /packages/realm/bindgen/vendor/bindgen-lib/generated/
 /packages/realm/generated/
+/packages/realm/src/generated/
```

Comment:
I needed to move `core.ts` into `src` when adopting TypeScript projects, because of:

> error TS6059: File '/Users/kraen.hansen/Projects/realm-js/packages/realm/generated/core.ts' is not under 'rootDir' '/Users/kraen.hansen/Projects/realm-js/packages/realm/src'. 'rootDir' is expected to contain all source files.
  The file is in the program because:
    Matched by include pattern '../generated/core.ts' in '/Users/kraen.hansen/Projects/realm-js/packages/realm/src/tsconfig.json'

And I don't really consider the wrapper .mjs files and realm.node file as "src" as it's not TS and won't be moved to "dist" by tsc.

Reply:
Hmm. I'm not sure we have a clear rule for what `src/` is. Is it where we put the "source code" for the realm package, or is it only the input for `tsc`? I had assumed the former, but it seems like you intend for it to be the later. I guess I don't feel too strongly either way, but we should be clear what our rule is.